app/controllers/users_controller.rb

```diff
@@ -8,7 +9,9 @@ def index
   def show
     # load the unpublished article for a particular page.
     @unpub_articles = Article.where(page: current_user.page, published: false)
-    @new_articles = Article.where(aasm_state: 'new', published: false).order("articles.page ASC")
+    @new_articles = Article
+                      .where(aasm_state: 'new', published: false)
```

Comment:
Since you're touching this line, AASM provides some [automatic scopes](https://github.com/aasm/aasm#automatic-scopes), so this line could be `Article.new.where(published: false).order(:page)`.